### PR TITLE
Replaced all the css into tailwind css on FontSection component

### PR DESCRIPTION
**Pull request by subhajit20**
# Description of changes

Replaced all the css into tailwind css on FontSection component
Issue - #1725 

# Issue Resolved

Fixes #NA

### CODE REVIEW

File: components/Branding/FontSection/FontSection.module.css [hunk 1]
```diff
@@ -1,12 +1,3 @@
...
-  list-style: none;
-  width: 100%;
-}
-
-.fontsList > li {
-  margin: 1rem;
-}
-
 .primaryFontFamily div > h6,
 .primaryFontFamily div > p {
```
Comment (subhajit20) [inline components/Branding/FontSection/FontSection.module.css:2]: I did not find these two classes.

File: components/Branding/FontSection/FontSection.module.css [hunk 2]
```diff
@@ -1,18 +0,0 @@
...
-  width: 100%;
-}
-
-.fontsList > li {
-  margin: 1rem;
-}
-
-.primaryFontFamily div > h6,
-.primaryFontFamily div > p {
-  font-family: var(--primaryFontFamily), sans-serif;
```
Comment (kylemh) [inline components/Branding/FontSection/FontSection.module.css:12]: Nice job on the condition.

Do you see here how it's 2 values separated by a comma?

1. var(--primaryFontFamily)
2. sans-serif

That's what's missing.

Comment (subhajit20) [reply components/Branding/FontSection/FontSection.module.css:12]: where to set that variable?

Comment (subhajit20) [reply components/Branding/FontSection/FontSection.module.css:12]: will it look like this ? -> 
fontFamily:{
   sans:{
      primaryFontFamily:'"DIN Condensed Bold"'
   }
}

Comment (kylemh) [reply components/Branding/FontSection/FontSection.module.css:12]:
```
fontFamily: {
  primary: "DIN Condensed Bold"
},
```

Comment (subhajit20) [reply components/Branding/FontSection/FontSection.module.css:12]: got it

Comment (subhajit20) [reply components/Branding/FontSection/FontSection.module.css:12]: done certain changes

**Review by kylemh — APPROVED** [review]

**Pull request merged**